Update  dependency map navigator (#19458)
DataDog/documentation: content/en/tracing/services/resource_page.md

## Dependency Map with Navigator

> You can also view a map of all of a resource's upstream and downstream service dependencies. With the Dependency Map Navigator, you can quickly see the flow of services with spans that go through the specific resource (such as endpoints or database queries) end-to-end, along with their request count. The map is based on a sample of ingested spans based on fixed sampling algorithm that considers the structure of traces. The sampling alforithm is not configuration and won't be impacted by ingestion control. The dependency map is only available for resources containing service entry spans. 

Reviewer: ```suggestion
You can also view a map of all of a resource's upstream and downstream service dependencies. With the Dependency Map Navigator, you can see the flow of services, with spans that go through a specific resource ( endpoint, database query, etc.) end-to-end, along with their request counts. 

This map is based on a sample of ingested spans; the sample is drawn by a fixed sampling algorithm that considers the structure of traces. The sampling algorithm is not configurable and is not impacted by ingestion control. 

The dependency map is only available for resources containing service entry spans. 
```
